airflow/settings.py

```diff
@@ -441,6 +463,9 @@ def configure_orm(disable_connection_pool=False, pool_class=None):
 
     global Session
     global engine
+    global async_engine
+    global create_async_session
```

Comment:
```suggestion
```

Reply:
why @ashb ?

Reply:
i think it's necessary ... it doesn't get set until configure_orm is called